resources/views/partials/sidebar.blade.php

```diff
@@ -1,42 +1,49 @@
-<div class="bg-magenta-800 text-white w-80 flex flex-col flex-shrink-0">
-    <div class="h-20 flex items-center px-4 mb-8">
-        <img src="{{ asset('images/university-logo.png') }}" alt="DU Logo" class="h-12 mr-3">
-        <a href="/" class="inline-block logo leading-tight max-w-sm">
-            <h1 class="text-lg font-bold">Department of <br> Computer Science</h1>
-        </a>
+<sidebar-nav inline-template>
+    <div v-if="is_visible" class="bg-magenta-800 text-white sm:w-80 flex flex-col flex-shrink-0 w-full">
```

Comment:
Instead of using inline templates, we can use default slots and wrap the contents inside a transition component for smooth transition.

Reply:
how to do that, we can't pass data from component to slot.

Reply:
We can do that. We can pass data to slots, you might not know about `scoped slots`. read about it (https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots)[here] on vue docs